insights/terraform/modules/bucket/main.tf

```diff
@@ -0,0 +1,39 @@
+data "aws_caller_identity" "current" {}
+
+resource "aws_s3_bucket" "main" {
+  bucket    = "bfd-insights-${var.sensitivity}-${data.aws_caller_identity.current.account_id}"
+  acl       = "private"
+  tags      = merge({sensitivity = var.sensitivity}, var.tags)
+
+  server_side_encryption_configuration {
+    rule {
+      apply_server_side_encryption_by_default {
+        sse_algorithm     = "AES256"
```

Comment:
Any reason to go with AES over KMS?

Reply:
AES is uses the KMS, so fundementally the security offered is the same. From a configuration point of view, it is difficult and more complex to get all the pieces to use a CMK stored in KMS. I couldn't get it to work on with Athena, for example.

Reply:
Look at the new changes. Switched to a CMK.